.github/workflows/pr.yml

```diff
@@ -130,7 +130,9 @@ jobs:
     - name: Install GraalVM native image
       run: gu install native-image
     - name: Build with Gradle
-      run: gradle nativeImage
+      run: |
+        ln -s isthmus-cli/proxies.json
```

Comment:
The macos builder is failing because it is looking in the wrong directory for this file.  Not sure why, but to work around the problem, I've added a symlink to where it's looking.  Just to get the build going again pending a 'proper' fix